### ISSUE TITLE
[BUG] Focused state in App Search is gone

### ISSUE THREAD

**Issue by validcube** [Deletescape-Media/Lawnchair#4339]
### Describe the bug

In `12.1-dev` and `13-dev` there was Focused state for selected or highlighted application

![image](https://github.com/LawnchairLauncher/lawnchair/assets/93124920/0c778434-5ac1-4d06-aa08-7afda93fe295)

In `14-dev` this is absent

![image](https://github.com/LawnchairLauncher/lawnchair/assets/93124920/aa489871-d8ad-4f68-ae36-a2034b421903)

Which caused slightly degraded in accessibility, the app should be highlighted and you can press tabs to cycle through applications

### Steps to reproduce

1. Open App Search
2. Search any application
3. See the focused state not being applied on every search

### Expected behavior

Application should have focused state appearance visible to user

### Device information

- Device: SDK 34 Emulated (Pixel 3a skin)
- OS: SDK 34 Emulated
- App version: Lawnchair 14 Beta 2

### Additional context

As for as I know this isn't capturable in Logcat

**Comment by MrSluffy** [comment]
@SuperDragonXD Can you check this thanks
